### PR TITLE
Add-1346 check if its double exist

### PR DESCRIPTION
**Pull request by Rwatana**
On this branch, It has some problems. So, I remake the folder.
Please check my code.
I revised the checkpoints.

### CODE REVIEW

File: 1351_count_negative_numbers_in_a_sorted_matrix.py [hunk 1]
```diff
@@ -1,18 +1,16 @@
+# T(n) O(1)
```
Comment (ryujumoto) [inline 1351_count_negative_numbers_in_a_sorted_matrix.py]: #time: O()
#space: O()

**Comment by Rwatana** [comment]
I revised the formats. Please check.

**Comment by ryujumoto** [comment]
Refer [this code](https://github.com/okuda-seminar/leetcode_answers/blob/main/1346_check_if_n_and_its_double_exist.py).

**Comment by Rwatana** [comment]
I follow the code and understand it.

**Review by ryujumoto — APPROVED** [review]

**Pull request merged**